src/util/get-files.ts

```diff
@@ -325,16 +359,16 @@ export async function docker(
 
     const ignoredFiles = clearRelative(
       dockerIgnore === null
-        ? await maybeRead(resolve(path, '.gitignore'))
+        ? await maybeRead(resolve(path, '.gitignore'), '')
         : dockerIgnore
     );
-
-    const filter = (dockerIgnore === null ? ignore : dockerignore)()
+    const ignoreInit = (dockerIgnore === null ? ignore : dockerignore) as any as typeof ignore;
```

Comment:
I thought I fixed this but the types are too different. Instead we favor types for `ignore` and assume `dockerignore` types are the same.